R/meteo_utils.r

```diff
@@ -54,27 +56,64 @@ meteo_coverage <- function(meteo_df,
 
   dplyr::group_by(meteo_df, id) %>%
     dplyr::do({
+      # calculate the date range for each station id
       rng <- range(.$date)
+      # dataframe with only start & end dates
       dat <- data.frame(start_date = rng[1],
                         end_date = rng[2],
                         total_obs = nrow(.), stringsAsFactors=FALSE)
+
       if (verbose) cat(sprintf("Station Id: %s\n", .$id[1]))
```

Comment:
Please use `message()` instead of `cat()` - allows user to suppress if not needed. yes, I realize you have a parameter for that, but if someone uses a script where verbose is hard coded, then you still want to allow messages to be suppressed as needed